apps/README.md

```diff
@@ -65,3 +66,19 @@ CONFIGURED_APPS += examples/hello
 endif
 ```
 
+## Suggestion for optimizing SmartFs Read
+
+Using just any buffer size for reading the contents of a SmartFs file, back-to-back(looped),
+might cause repetitive reading of sectors and performance degradation.
+Hence, programmers are encouraged to use an optimal buffer size for reading back-to-back from
+a SmartFs file.
+
+```
+stat("\mnt", &st);
+buf_size = st.st_blksize;
+```
+
```

Comment:
It's good approach to update readme document for this application.

Reply:
Thankyou